### PR TITLE
Small KubeLB Documentation improvements 2024-09

### PR DESCRIPTION
**Pull request by toschneck**
- add info about secret name more explicit
- add info about internal IP usage

### CODE REVIEW

**Comment by toschneck** [comment]
Some of the changes have already be addressed, now just smaller adjustments.

File: content/kubelb/v1.1/installation/tenant-cluster/_index.en.md [hunk 1]
```diff
@@ -84,7 +97,7 @@ kubelb:
 ### Install the helm chart
 
 ```sh
-helm pull oci://quay.io/kubermatic/helm-charts/kubelb-ccm-ee --version=v1.1.2 --untardir "." --untar
```
Comment (ahmedwaleedmalik) [inline content/kubelb/v1.1/installation/tenant-cluster/_index.en.md:87]: Can you please remove the change for image tag? 

`s/v1.1.1/v1.1.2`

**Review by ahmedwaleedmalik — CHANGES_REQUESTED** [review]

**Comment by toschneck** [comment]
@ahmedwaleedmalik resolved it, but I think long term it would be good to create a latest tag
kubelb-ccm-ee · Quay 2024-12-04 00-56-43

**Review by ahmedwaleedmalik — APPROVED** [review]
/approve

**Pull request merged**